plugins/Chart/__init__.py

```diff
@@ -1,48 +1,76 @@
 from discord.ext import commands
 from Plugin import AutomataPlugin
 import discord
-import matplotlib.pyplot as plt
 import io 
 from typing import List
+import requests
+import datetime
 
-def parseArguments(args: str) -> List[float]:
+# ==== UTIL FUNCTIONS =====
+# TODO: Write unit tests
+
+def parseArguments(args: str) -> List[str]:
     """ Takes the arguments from barplot function 
-        And converts it to a list of usable numbers. """
+        And converts it to a list of string numbers. """
     
     if "," in args:
         temp = args.strip().split(",")
         return 
     else:
         temp =  args.strip().split()
     
-    return list(map(float, temp))
+    # Temp to ensure that all parameters are numbers
+    temp = list(map(float, temp))
+    return list(map(str, temp))
+
+def getErrorEmbed(title="Error Title", message="Error Message") -> discord.Embed:
+    """
+    Generate an embed for error messages
+    """
+    emb = discord.Embed(
+        title=message,
+        timestamp=datetime.datetime.now(),
+        colour = discord.Colour(0xEF440E)
+    )
+    emb.set_author(
+        name=title,
+        icon_url="https://i.imgur.com/UjdPxZw.png" # Potentially is not the safest. TODO: Find alternative solution
```

Comment:
You could host this in your own github account, or something similar. Like Vibe plugin - https://github.com/MUNComputerScienceSociety/Automata/blob/73085f480e48a05a5e5422ec22c2f5e798d62706/plugins/Vibe/__init__.py#L6